cmd/utils.go

```diff
@@ -1072,6 +1072,13 @@ func getStackLabels(config *RootCommandConfig) (map[string]string, error) {
 		}
 	}
 
+	imageAndDigest := data[0]["RepoDigests"].([]interface{})
+	if len(imageAndDigest) > 0 { //Check that the image has a digest
+		imageAndDigestStr := fmt.Sprintf("%v", imageAndDigest[0])
+		digest := strings.Split(imageAndDigestStr, "@")
+		labels[appsodyStackKeyPrefix+"digest"] = digest[1]
```

Comment:
Do we want the `sha256:` prefix in the digest?

Reply:
@Kamran64 apparently we need to add the `sha256:` prefix.. sorry!!

Reply:
@EnriqueL8 Np, readded in the latest commit